docs/sources/enterprise/saml.md

```diff
@@ -204,6 +204,61 @@ org_mapping = Engineering:2, Sales:3
+1. Choose **Web** as a platform.
+1. Select **SAML 2.0** in the Sign on method section.
+1. Click **Create**.
+1. On the **General Settings** tab, enter a name for your Grafana integration and optionally upload a logo.
+1. On the **Configure SAML** tab, enter the SAML information related to your Grafana instance:
+    - In the **Single sign on URL** field, use the `/saml/acs` endpoint URL of your Grafana instance, for example, `https://grafana.example.com/saml/acs`.
+    - In the **Audience URI (SP Entity ID)** field, use the `/saml/metadata` endpoint URL, for example, `https://grafana.example.com/saml/metadata`.
+    - Leave the default values for **Name ID format** and **Application username**.
+    - In the **ATTRIBUTE STATEMENTS (OPTIONAL)** section, enter the SAML attributes to be shared with Grafana, for example:
+      
+      | Attribute name (in Grafana) | Value (in Okta profile)                |
+      | --------------------------- | -------------------------------------- |
+      | Login                       | `user.login`                           |
+      | Email                       | `user.email`                           |
+      | DisplayName                 | `user.firstName + " " + user.lastName` |
+    
+    - In the **GROUP ATTRIBUTE STATEMENTS (OPTIONAL)** section, enter a group attribute name (for example, `Group`) and set filter to `Matches regex .*` to return all user groups.
+1. Click **Next**.
+1. On the final Feedback tab, fill the form and click **Finish**.
+
+Once application is created, configure Grafana to use it for SAML authentication.
+
+1. In the `[auth.saml]` section in the grafana configuration file, set [`enabled`]({{< relref "./enterprise-configuration.md#enabled" >}}) to `true`.
+1. Configure [certificate and private key]({{< relref "#certificate-and-private-key" >}}).
+1. On the Okta application page you redirected after application created, navigate to the **Sign On** tab and find **Identity Provider metadata** link in the **Settings** section.
+1. Set the [`idp_metadata_url`]({{< relref "./enterprise-configuration.md#idp-metadata-url" >}}) to the URL obtained from the previous step. This URL should look like `https://<your-org-id>.okta.com/app/<application-id>/sso/saml/metadata`.
+1. Set the [`assertion_attribute_login`]({{< relref "./enterprise-configuration.md#assertion-attribute-login" >}}), [`assertion_attribute_email`]({{< relref "./enterprise-configuration.md#assertion-attribute-email" >}}), [`assertion_attribute_name`]({{< relref "./enterprise-configuration.md#assertion-attribute-name" >}}), and [`assertion_attribute_groups`]({{< relref "./enterprise-configuration.md#assertion-attribute-groups" >}}) options to the attribute names configured at the **step 10** of the SAML integration setup. You can find this attributes on the **General** tab of the application page (**ATTRIBUTE STATEMENTS** and **GROUP ATTRIBUTE STATEMENTS** in the **SAML Settings** section).
```

Comment:
```suggestion
1. Set the following options to the attribute names configured at the **step 10** of the SAML integration setup. You can find this attributes on the **General** tab of the application page (**ATTRIBUTE STATEMENTS** and **GROUP ATTRIBUTE STATEMENTS** in the **SAML Settings** section). 
  - [`assertion_attribute_login`]({{< relref "./enterprise-configuration.md#assertion-attribute-login" >}})
  -  [`assertion_attribute_email`]({{< relref "./enterprise-configuration.md#assertion-attribute-email" >}})
  - [`assertion_attribute_name`]({{< relref "./enterprise-configuration.md#assertion-attribute-name" >}})
  - [`assertion_attribute_groups`]({{< relref "./enterprise-configuration.md#assertion-attribute-groups" >}}) 
```

Reply:
Hopefully my indentation is right, but double check.

Reply:
I use 4 spaces because 2 is working a bit strange (not enough indentation to the left), but 4 looks more consistent.